src/bin.ts

```diff
@@ -9,14 +10,14 @@ if (!str) {
 const initUrl = new URL(str);
 const results = new Map<string, AwakenResult>();
 let isSuccessful = true;
-const onAwaken = ({ status, url, referer, msg }: AwakenResult) => {
-  if (200 <= status && status <= 299) {
+const onAwaken = ({ status, url, referer, msg }: AwakenResult): void => {
+  if (status >= 200 && status <= 299) {
```

Comment:
Bummer. I kinda like the old format since it looks like BETWEEN. But not a big deal 🙂